### ISSUE TITLE
Mention "as a handy tool for analytical and presentation task" in .md

### ISSUE THREAD

**Issue by the-liquid-metal** [vzhd1701/gridplayer#51]
Thank you Vlad for your effort to push VridPlayer to the higher level.

I believe that GridPlayer deserve a place of analytical and presentational tool, not just a video player. Thought some features needed to implement: #19 (adjustment and effect) and #24 (border on selected movies).

In the past, i doubt that some features would not be approved due to lact of your comment. But I was wrong.

Now people on many fields can use this app on their laboratory or class room. Take biology for example. People can compare behavior or process of their anaimals or plants.

**Comment by vzhd1701** [comment]
Thank you for the commendation. Sorry that I left you without updates for so long. I'll implement the features you mentioned.

Do you mean I should list more practical use cases in the readme?

**Comment by the-liquid-metal** [comment]
> 
> Do you mean I should list more practical use cases in the readme?

Yes please. One or two fields may be. The more descriptive the better, in order to attract people to try this app.

**Comment by the-liquid-metal** [comment]
In case you want to collect whatever user use this app on their fields:
create an issue which open forever, and then inform people to tell their story on that issue.
You can update the .md with most inspiring story.

**Comment by the-liquid-metal** [comment]
Add tag to this repository: "video-analitical-tool"
